app/controllers/flood_risk_engine/enrollments/steps_controller.rb

```diff
@@ -30,10 +30,18 @@ def update
         end
       end
 
+      def remove_exemption
+        @enrollment = Enrollment.find(params[:id])
+        @exemption = Exemption.find(params[:exemption_id])
+        enrollment.exemptions.destroy(@exemption)
+        step_back if enrollment.exemptions.empty?
+        redirect_to step_url
+      end
+
```

Comment:
Yes - I started with using a patch method on the link, but then realised that if JavaScript was turned off, it would become a GET. So rather than handling that as a separate case, I made it the main behaviour.